### DISCUSSION TITLE
WebGIS6 Desktop - Editieren (Oracle)

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#193] [Q&A]
Hallo Jürgen!

Um via WebGIS direkt in die Oracle-DB (WIS) zu editieren, wurde im WebGIS6 (und WebGIS5) die Konfig via Inhaltsverzeichnis und TocName vorgenommen und dazu noch im Konfiguration/etc/editing-Verzeichnis eine workspace.xml und unter themes eine edit_wis.xml angelegt.
![image](https://github.com/user-attachments/assets/b01cae13-e201-4d17-b9c1-f22cc01c065a) ![image](https://github.com/user-attachments/assets/992ae240-e6e6-4b24-a7df-5d55c307dbaf)

Das Editieren geht damit sowohl im WebGIS6 als auch im WebGIS5 einwandfrei. 

Im WebGIS6 im Mobile (Touch-Bedienung)-Layout funktioniert das Anlegen/Bearbeiten/Löschen von Objekten wie gewohnt. 
![image](https://github.com/user-attachments/assets/71a31bfb-8a98-4ad1-9747-169cfb4e064b)

Im WebGIS6 Desktop (Tastatur/Maus)-Layout funktioniert nur noch das Anlegen von Objekten. Ich habe im Moment keine Möglichkeit das Objekt zu bearbeiten bzw. zu löschen.
![image](https://github.com/user-attachments/assets/3cbfc740-9972-4d19-ae5f-4a2ad75a8133)

Ist das Verschieben/Löschen von Objekten, die direkt in eine Oracle-DB editiert werden, auch im Desktop-Layout möglich? Wie müsste die Konfig angepasst werden, um alle drei Editieroptionen verfügbar zu haben?

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Die Werkzeuge **Bestehendes Objekt bearbeiten** und **Bestehendes Objekt löschen** gibt es in der WebGIS 6 Desktop Edit Umgebung gar nicht mehr. Hier ist der Weg so:

1. Ein Objekt selektieren: Entweder mit der *Punkt* oder *Rechteck* Selektion. Oder einfach über das *Identify* Werkzeug.
2. In der Tabelle werden dann die Werkzeug zum bearbeiten angeboten

Oder ist da bei euch im Oracle-DB etwas anders?

**Comment by Andreas-Lampert** [comment]
Hallo Jürgen!

Ich hatte keine Abfragen konfiguriert (war mit dem Mobilen-Layout nicht notwendig). Ich habe jetzt für die Editier-Layer eine Abfrage konfiguriert und damit kann das bestehende Objekt bearbeitet bzw. gelöscht werden. 
Ich schließe das Ticket somit. 

Lg, Andi
